subsys/net/lib/lwm2m/lwm2m_obj_connmon.c

```diff
@@ -39,10 +42,9 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define CONNMON_CELLID				8
 #define CONNMON_SMNC				9
 #define CONNMON_SMCC				10
-#if defined(CONFIG_LWM2M_CONNMON_OBJECT_VERSION_1_2)
```

Comment:
Would be nice to have separate comments here, as in the original proposal by @SeppoTakalo (`/* Object version 1.2 */` etc).

Reply:
@rlubos :  I have pushed a new version. Can you please check this? Thx